### ISSUE TITLE
--paper-input-container-focus-color should not be set

### ISSUE THREAD

**Issue by kito99** [ellipticaljs/paper-autocomplete#51]
Currently, paper-autocomplete sets --paper-input-container-focus-color to a specific value:

```css
      --paper-input-container-focus-color: #2196f3;
```

This value shouldn't be hardcoded, because it's set to `var(--primary-color)` by default. As a result, paper-autocomplete won't match other paper-input derived fields on the same page.

This is easy to work-around by adding some CSS in the parent element:

```css
			paper-autocomplete {
				--paper-input-container-focus-color: var(--primary-color);
			}
```

However, I don't see why the value should be hardcoded in the first place.

**Comment by tachyon1337** [comment]
@kito99 

No, it shouldn't be. But the component was written around the time of Polymer 1.0, with css vars being brand new and so forth, and I hard coded a value that no one hitherto has complained about. But yes we will update that to be a var.

**Comment by jhuesos** [comment]
Fixed by #56 in `v2.3.1`
